auction-app-backend/src/main/java/com/atlantbh/auctionappbackend/domain/User.java

```diff
@@ -0,0 +1,157 @@
+package com.atlantbh.auctionappbackend.domain;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity(name = "User")
+@Table(
+        name = "app_user",
+        uniqueConstraints = {
+                @UniqueConstraint(name = "user_email_unique", columnNames = "email")
+        }
+)
+public class User {
+    @Id
+    @SequenceGenerator(
+            name = "app_user_sequence",
+            sequenceName = "app_user_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "app_user_sequence"
+    )
+    @Column(updatable = false)
+    private Long id;
+
+    @Column(nullable = false)
+    private String firstName;
+
+    @Column(nullable = false)
+    private String lastName;
+
+    @Column(nullable = false)
+    private String email;
+
+    @Column(nullable = false)
+    private String password;
+
+    @Column(nullable = false)
+    private String role;
+
+    private Long streetId;
```

Comment:
Since we still do not have a `street` table, we can remove this property.

Reply:
Sure.